hles/extraction/src/test/scala/org/broadinstitute/monster/dap/CslbExtractionPipelineBuilderIntegrationSpec.scala

```diff
@@ -0,0 +1,75 @@
+package org.broadinstitute.monster.dap
+
+import java.time.{LocalDate, LocalTime, OffsetDateTime, ZoneOffset}
+
+import better.files.File
+import org.broadinstitute.monster.common.PipelineBuilderSpec
+
+class CslbExtractionPipelineBuilderIntegrationSpec extends PipelineBuilderSpec[Args] {
+  val outputDir = File.newTemporaryDirectory()
+  val cslbOutputDir = File(outputDir, CslbExtractionPipeline.subdir)
+  override def afterAll(): Unit = outputDir.delete()
+
+  // We know that some records were updated in this time range, so it should
+  // be fine to pull consistently without worrying about changes in data size.
+  val start =
+    OffsetDateTime.of(LocalDate.of(2020, 11, 15), LocalTime.MIDNIGHT, ZoneOffset.UTC)
+
+  val end =
+    OffsetDateTime.of(LocalDate.of(2020, 11, 16), LocalTime.MIDNIGHT, ZoneOffset.UTC)
```

Comment:
also finding myself worrying that these timestamps are more specific than they need to be, since their value isn't relevant.

Reply:
Property based testing might be a thing we want to investigate, but I think these dates are fine for now.